include/alpaka/intrinsic/Traits.hpp

```diff
@@ -77,5 +84,51 @@ namespace alpaka
                 intrinsic,
                 value);
         }
+
+        //-----------------------------------------------------------------------------
+        //! Returns the 1-based position of the least significant bit set to 1
+        //! in the given 32-bit value. Returns 0 for input value 0.
+        //!
+        //! \tparam TIntrinsic The intrinsic implementation type.
+        //! \param intrinsic The intrinsic implementation.
+        //! \param value The input value.
+        ALPAKA_NO_HOST_ACC_WARNING
+        template<
+            typename TIntrinsic>
+        ALPAKA_FN_ACC auto ffs(
+            TIntrinsic const & intrinsic,
+            int value)
+        -> int
+        {
+            using ImplementationBase = concepts::ImplementationBase<ConceptIntrinsic, TIntrinsic>;
+            return traits::Ffs<
+                ImplementationBase>
+            ::ffs(
+                intrinsic,
+                value);
+        }
+
+        //-----------------------------------------------------------------------------
+        //! Returns the 1-based position of the least significant bit set to 1
+        //! in the given 64-bit value. Returns 0 for input value 0.
+        //!
+        //! \tparam TIntrinsic The intrinsic implementation type.
+        //! \param intrinsic The intrinsic implementation.
+        //! \param value The input value.
+        ALPAKA_NO_HOST_ACC_WARNING
+        template<
+            typename TIntrinsic>
+        ALPAKA_FN_ACC auto ffs(
+            TIntrinsic const & intrinsic,
+            long long value)
+        -> int
+        {
+            using ImplementationBase = concepts::ImplementationBase<ConceptIntrinsic, TIntrinsic>;
+            return traits::Ffs<
+                ImplementationBase>
+            ::ffs(
```

Comment:
should this not be `ffsll()` and require the incude `<ctsring>` in this file see https://linux.die.net/man/3/ffsll

Reply:
I followed the same logic as popcount: alpaka provides a unified interface (be it with templates or overloading like we discussed during review of #1004), without this ugly C postfixes